build.sh

```diff
@@ -10,5 +10,5 @@ if [[ "$1" -gt "-1" ]] && [[ $1 == "dev" ]]; then
     packer build -var-file=photon-builder.json -var-file=photon-version.json photon-dev.json
 else
     echo "Applying packer build to photon.json ..."
-    packer build -var-file=photon-builder.json -var-file=photon-version.json photon.json
+    packer build -var "VEBA_VERSION=$(cat VERSION)" -var "VEBA_COMMIT=$(git rev-parse --short HEAD)" -var-file=photon-builder.json -var-file=photon-version.json photon.json
```

Comment:
The only concern I have is that the embedded variable `VEBA_COMMIT` might resolve to a stale commit, e.g. dirty git repository where you make changes but those have not been committed. A typical workaround is to check for a clean git repo and otherwise fail the build to make sure all changes are committed (or stashed). This is enforced in the build of the event router for example.

Otherwise LGTM